### ISSUE TITLE
Support For OpenRTB 2.6-202409

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#3762]
OpenRTB 2.6-202407 contains a few of additions to OpenRTB that Prebid Server should support in the JSON schema validation. 

https://github.com/InteractiveAdvertisingBureau/openrtb2.x/pull/103

- user.eids[].inserter - string
- user.eids[].matcher - string
- user.eids[].mm - int

No special logic or validations. When an eids object is passed to a bid adapter, these are just additional data. If an eids object is removed for any reason, these values would be removed with it.

**Comment by SyntaxNode** [comment]
The changes to the EID object as part of OpenRTB 2.6-202407 (not yet merged) is designed to provide sellers with better clarity on the buyerid. This [ID Generation Provenance and Clarification](https://iabtechlab.com/id-generation-provenance-and-clarification/) IAB article discussing the objective, also known as ID Bridging.

There is new logic needed in Prebid Server to support this object. When the buyerid is inserted by Prebid Server as the result of cookie syncing, Prebid Server is expected to add a new eid object with the inserter and (likely) matcher identifying the host company and mm (match method) set to 2 / Browser Cookie Sync.

The inserter should match the host company's ads.txt domain. I'm less certain on if it's appropriate or necessary for Prebid Server to set the matcher.

**Comment by bretg** [comment]
Note: as discussed on Friday, split out OpenRTB 2.6-202402 into a separate issue.

**Comment by bretg** [comment]
Discussed in committee. There's an open question about how we should set eid.source -- all we have is the cookie family (normally biddercode), not a domain.

Maybe we need to add a new field to the YAML file to specify?

```
    usersync:
      cookie-family-name: bidderA
      eids-source: bidderA.co.uk
      iframe:
        url:
      redirect:
        url:
```

Maybe if `eids-source` isn't specified, it defaults to BIDDERCODE.com? That could be dangerous. Maybe better to just not create an EID entry fro them if not specified.

An example:

```
  "user": {
      "eids": [
        {
          "source": "CONFIGURED-EID-SOURCE", 
          "inserter": "hostcompany.com",
          "mm": 2,
          "uids": [
            {
              "id": "ID-FROM-UIDS-COOKIE",
              "atype": 1
            }
          ]
        },
```

**Comment by bretg** [comment]
This was discussed in the identity committee last week. We changed the proposed default from BIDDERCODE.com to being derived from URLSYNCDOMAIN

**Comment by bretg** [comment]
@patmmccann - do you have a sense for what's worse here: the wrong eid.source or no entry at all?

**Comment by bretg** [comment]
Looking to move this to ready-for-dev

**Comment by bretg** [comment]
Also adding @jdwieland8282 - would like to bring this one to a conclusion.

**Comment by jdwieland8282** [comment]
The identity committee believes that it is not necessary to copy the output of a traditional user sync to the eids array. Identifiers derived from a user sync may continue to be used in the `user.buyeruid`.

**Comment by bretg** [comment]
As the Identity Committee suggests, this issue now just tracks expanding the schema to allow these new fields.

They also noted that interested community members could build this if they insist -- the technical details of the proposal are ok but that PBS Host Companies would require opt-in configuration for whether the EIDs were added.

**Comment by bretg** [comment]
Done with PBS-Java 3.13

**Comment by SyntaxNode** [comment]
Implemented in PBS-Go v3.5.0.
